models/policies.go

```diff
@@ -28,23 +36,30 @@ type Policy struct {
 	Conditions []Condition `bson:"conditions" json:"conditions,omitempty"`
 }
 
-type NewPolicy struct {
+type UpdateResult struct {
+	ModifiedCount int
+	UpsertedCount int
```

Comment:
Should this model also have JSON tags?

Reply:
Its an internal struct used to return results from repository layer to api layer, so we dont need a JSON tag I think.